File: base-v3/src/test/java/com/adyen/checkout/base/models/TestOutputData.java
/*
 * Copyright (c) 2019 Adyen N.V.
 *
 * This file is open source and available under the MIT license. See the LICENSE file for more info.
 *
 * Created by arman on 12/4/2019.
 */

package com.adyen.checkout.base.models;

import com.adyen.checkout.base.component.data.output.OutputData;

public class TestOutputData implements OutputData {

    public String type;
    public boolean isValid;

    public TestOutputData(String type) {
        this.type = type;
    }

    public TestOutputData(String type, boolean isValid) {
        this.type = type;
        this.isValid = isValid;
    }

    @Override
    public boolean isValid() {
        return this.isValid;
    }


}

File: components-core/src/test/java/com/adyen/checkout/components/models/TestOutputData.kt
/*
 * Copyright (c) 2019 Adyen N.V.
 *
 * This file is open source and available under the MIT license. See the LICENSE file for more info.
 *
 * Created by arman on 12/4/2019.
 */

package com.adyen.checkout.components.models;

import com.adyen.checkout.components.base.OutputData;

public class TestOutputData implements OutputData {

    public boolean isValid;

    public TestOutputData() {
        this(false);
    }

    public TestOutputData(boolean isValid) {
        this.isValid = isValid;
    }

    @Override
    public boolean isValid() {
        return this.isValid;
    }


}
